### ISSUE TITLE
Apply page page accessibility issues

### ISSUE THREAD

**Issue by gixxerblade** [Vets-Who-Code/vets-who-code-app#223]
# Accessibility

## Form elements do not have associated labels

Labels ensure that form controls are announced properly by assistive technologies, like screen readers.

### Failing Elements

- `<input type="text" class="form-control input-lg" id="lastName" name="lastName" placeholder="Last Name">`
- `<input class="form-control input-lg" id="email" name="email" placeholder="Email">`

**To Reproduce**
1. Go to 'https://vetswhocode.io/apply'
2. Click on 'Developer Tools (F12)'
3. Click on Lighthouse
4. Generate report

**Expected behavior**
Add `aria-label`'s to the inputs. Assistive technology users rely on these labels to navigate forms. Mouse and touchscreen users also benefit from labels because the label text makes a larger click target.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by emma-r-slight** [comment]
I have noticed that while you can easily navigate through the page with tab on Chrome,  Firefox just moves between the top menu bar of the page then focus moves to the browser application- URL bar, back, refresh etc.  Similar problem with Safari.
